tf-modules/cumulus-rds-tf/provision_database

```diff
@@ -0,0 +1 @@
+../../lambdas/db-provision-user-database/dist/webpack/
```

Comment:
This is a representation of a symlink - this allows the module to reference the lambda source locally whilest the build script can package the actual zip file in the proper location for the released module.

Reply:
This might be easier as a short meet/walkthrough.  

That said there *is* a symlink in the repo, this is how github is representing it in the diff.    The file here is *actually* a symlink in the git repo. 

>How does this work for the release artifact? It's still pointing to ../../lambdas/db-provision-user-database/dist/webpack/ even in the release artifact, correct?

So the release artifact is zipped from the `dist` directory, it retains the same file structure (namely `./provision_database` contains the .zip of the lambda) but the .zip has been bundled directly into that path, whereas in development the symlink refers to the built file at the target of this symlink.